build/release/Makefile

```diff
@@ -0,0 +1,132 @@
+# Copyright 2016 The Rook Authors. All rights reserved.
```

Comment:
this is MUCH more succinct and consolidated than the release scripts were.  nice.